openmetadata-docs/content/deployment/upgrade/backup-metadata.md

```diff
@@ -152,3 +153,89 @@ Uploading dir1/dir2/openmetadata_202201250823_backup.sql to http://localhost:900
 If we now head to the minio console and check the `my-backup` bucket, we'll see our SQL dump in there.
 
 <Image src="/images/deployment/backup/minio-example.png" alt="minio"/>
+<br/>
+<br/>
+
+# Restore Metadata
+
+## Introduction
+
+SQL file which is generated using Backup metadata CLI
+can restore using Restore metadata CLI.
+
+## Requirements
+
+The restore CLI needs to be used with `openmetadata-ingestion` version 0.12.1 or higher.
+
+## Installation
+
+The CLI comes bundled in the base `openmetadata-ingestion` Python package. You can install it with:
+
+```commandline
+pip install openmetadata-ingestion
```

Comment:
All these steps are the same as for the backup CLI. We can remove the lines above

Reply:
Sure @pmbrull